fastestimator/estimator/estimator.py

```diff
@@ -38,6 +38,8 @@ def __init__(self,
         self.validation_steps = validation_steps
         self.traces = traces
         self.log_steps = log_steps
+        self.inputs = None
+        self.do_eval = None
```

Comment:
do these changes break any other components?

Reply:
pylint is complaining property defined outside __init__()

Reply:
why is do_eval changed from False to None? Wouldn't it affect other components of estimator?